src/app/components/dashboard/Dashboard.module.css

```diff
@@ -0,0 +1,54 @@
+.dashboard-wrapper {
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+
+  .dashboard-content {
+    display: flex;
+    flex: 1 1 auto;
+    flex-direction: column;
+    gap: 32px;
+    overflow: auto;
+    padding: 16px 32px 32px;
+
+    .dashboard-filter-area {
+      background: var(--color-beige-86);
+      border-radius: var(--border-radius-default);
+      display: flex;
+      gap: 16px;
+      padding: 16px;
+    }
+
+    .dashboard-two-column {
```

Comment:
will start to review this @amoedoamorim just adding a quick comment that I think if you add `flex-wrap: wrap;` to this, then we will get some nice reponsiveness on the dashboard